Source/Scripts/Syntax.cs-schema

```diff
@@ -50,6 +63,14 @@ abstract class Type : NodeWithOrigin
 {
 }
 
+abstract class TypeProxy : Type
```

Comment:
Why is this type here? This feels like something that's part of resolution.

Reply:
I agree, and I've removed it. (see also https://github.com/dafny-lang/dafny/pull/6163#discussion_r2023893414)